dashboard/public/pixelation/pixelation.html

```diff
@@ -0,0 +1,80 @@
+<link type="text/css" rel="stylesheet" href="/pixelation/pixelation.css">
+<script type="text/javascript" src="/pixelation/pixelation.js"></script>
```

Comment:
As long-term cleanup, I might move this script tag to the end of the file and bind all the onclick/onchange handlers from script.  For now, maybe we can just roll the contents of the script tag at the end of the page into a `main()` method, and call that?